test/built-ins/Iterator/prototype/Symbol.toStringTag/prop-desc.js

```diff
@@ -1,20 +1,17 @@
 // Copyright (C) 2023 Michael Ficarra. All rights reserved.
 // This code is governed by the BSD license found in the LICENSE file.
 /*---
-es6id: 25.1.2.1
```

Comment:
https://github.com/tc39/test262/blob/main/CONTRIBUTING.md#esid says `esid` is required for all new tests; I'm not sure how strictly that's enforced but might be nice to update this instead of removing it entirely

Reply:
Added.